src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -247,6 +255,11 @@ export function getPrismaClient(config: GetPrismaClientOptions): any {
         cwd = config.dirname
       }
 
+      const experimentalFeatures = config.generator?.experimentalFeatures ?? []
+      if (!experimentalFeatures.includes('aggregations')) {
```

Comment:
Why are we explicitly pushing aggregations and not other features?

Perhaps development skeleton remaining here?

Reply:
That's because in the client implementation we're always using that flag for now, as there was a breaking change in the underlying aggregations API and we don't want to ship 2 codebases. The feature flag will be called `aggregateApi` and I'm adding that now.

Reply:
That is interesting